app/assets/javascripts/client/inventories/analysisButton.js

```diff
@@ -0,0 +1,17 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient')
+      .directive('analysisButton', analysisButton);
```

Comment:
This is alright for now - we need to clean up the naming conventions around the platform.